src/main/java/org/candlepin/insights/controller/InventoryController.java

```diff
@@ -180,9 +184,11 @@ private void extractHypervisorFacts(Consumer consumer, Map<String, String> pinhe
     }
 
     private List<ConduitFacts> getValidatedConsumers(String orgId) {
+        String accountNumber = orgListStrategy.getAccountNumberForOrg(orgId);
         List<ConduitFacts> conduitFactsForOrg = new LinkedList<>();
         for (Consumer consumer : pinheadService.getOrganizationConsumers(orgId)) {
             ConduitFacts facts = getFactsFromConsumer(consumer);
+            facts.setAccountNumber(accountNumber);
```

Comment:
This is not actually getting applied to the facts that are sent to the server. If you look on line 195 we are foolishly getting a new set of facts before adding to the conduitFactsForOrg list.
```java
            if (violations.isEmpty()) {
                conduitFactsForOrg.add(getFactsFromConsumer(consumer));
            }
```
And should be:
```java
            if (violations.isEmpty()) {
                conduitFactsForOrg.add(facts);
            }
```
Looks like an old bug and we should be adding the facts instead. :)

Reply:
Good catch. Pushing update w/ a fix.

Reply:
Looks good here.